### ISSUE TITLE
Vast number of "No content roots with bsconfig.json file found" warnings in IJ log file

### ISSUE THREAD

**Issue by jfehrle** [reasonml-editor/reasonml-idea-plugin#249]
plugin version: 0.88.1-2020.1

I'm seeing a vast number of these messages in my IntelliJ log.  This is one second's worth.  Perhaps they can be eliminated or greatly reduced.  I'm using OCaml, not ReasonML.

```
2020-05-21 16:18:18,117 [614645233]   WARN -              ReasonML.platform - No content roots with bsconfig.json file found 
2020-05-21 16:18:18,117 [614645233]   WARN -              ReasonML.platform - No content roots with bsconfig.json file found 
2020-05-21 16:18:18,118 [614645234]   WARN -              ReasonML.platform - No content roots with bsconfig.json file found 
2020-05-21 16:18:18,118 [614645234]   WARN -              ReasonML.platform - No content roots with bsconfig.json file found 
2020-05-21 16:18:18,118 [614645234]   WARN -              ReasonML.platform - No content roots with bsconfig.json file found 
2020-05-21 16:18:18,119 [614645235]   WARN -              ReasonML.platform - No content roots with bsconfig.json file found 
2020-05-21 16:18:18,119 [614645235]   WARN -              ReasonML.platform - No content roots with bsconfig.json file found 
2020-05-21 16:18:18,119 [614645235]   WARN -              ReasonML.platform - No content roots with bsconfig.json file found 
```

**Comment by jfehrle** [comment]
It makes it impossible to find anything else in the log.  For example, my most recent IJ log file with thousands of lines covers only 67 seconds and these are _only_ messages present in the file.  It would be very hard to find any other messages in the file to debug non-ReasonML problems.

**Comment by giraud** [comment]
Fixed in master, I'll try to publish a new version very soon

**Comment by giraud** [comment]
should be fixed in 0.89
